public/src/js/constants/article-meta-fields.js

```diff
@@ -64,6 +64,30 @@ export default Object.freeze([
         type: 'text'
     },
     {
+        key: 'imageSource',
+        editable: true,
+        dropImage: true,
+        omitForSupporting: true,
+        visibleWhen: 'imageReplace',
+        label: 'replacement image URL',
+        validator: {
+            params: {
+                src: 'imageSrc',
+                width: 'imageSrcWidth',
+                height: 'imageSrcHeight',
+                origin: 'imageSrcOrigin',
+                imageSource: 'imageSource',
+                options: {
+                    maxWidth: 1000,
```

Comment:
Sure thing! Frontend uses CAPI's [`isMaster` field](https://github.com/guardian/frontend/pull/12254/files#diff-e349969fa55f6fac9fb629179b58be55R33)... Grid and cropper emits an element named `master` as opposed to other `assets`. But I wouldn't know. @kenoir would, though. He's the `/master/`'s master.